enigma-server/src/main/java/org/quiltmc/enigma/network/DedicatedEnigmaServer.java

```diff
@@ -134,9 +134,7 @@ public static void main(String[] args) {
 			return;
 		}
 
-		// noinspection RedundantSuppression
-		// noinspection Convert2MethodRef - javac 8 bug
```

Comment:
a while ago I asked earth what this meant, he said that it should be fine to toss